src/main/java/core/basesyntax/service/impl/WriteServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class WriteServiceImpl implements WriterService {
+    private static final String fileName = "src/main/resources/result.csv";
+
+    @Override
+    public void writeToFile() {
+        StringBuilder results = new StringBuilder("fruit,quantity" + System.lineSeparator());
+        Storage.fruits.entrySet().stream()
+                .forEach(f -> results.append(f.getKey()).append(",")
+                        .append(f.getValue()).append(System.lineSeparator()));
```

Comment:
We are inside WriteService but this is not writing. Do not mix different logic into one class.